editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5448,7 +5448,6 @@ CanvasItemEditor::CanvasItemEditor() {
 	main_menu_hbox->add_child(smart_snap_button);
 	smart_snap_button->set_toggle_mode(true);
 	smart_snap_button->connect(SceneStringName(toggled), callable_mp(this, &CanvasItemEditor::_button_toggle_smart_snap));
-	smart_snap_button->set_tooltip_text(TTR("Toggle smart snapping."));
```

Comment:
This tooltip was actually different from shortcut name, so it did show (unlike the ones above).

Reply:
Sure, it's _worded_ differently, but it says the same thing.